x-pack/test/task_manager_claimer_mget/test_suites/task_manager/check_registered_task_types.ts

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { Response as SupertestResponse } from 'supertest';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
```

Comment:
do you think we need to copy this test over? It's not testing anything with the task claimer and it would just be one more location to update when new tasks are added.

Reply:
hahahaha, I just copied ALL the files over and started running them, assuming there'd be a lot of errors and would figure stuff like this out.  Weren't many errors so I didn't actually look through the other tests!  Thx